internal/decision/blockstoremanager.go

```diff
@@ -26,24 +26,24 @@ func newBlockstoreManager(bs bstore.Blockstore, workerCount int) *blockstoreMana
 		bs:          bs,
 		workerCount: workerCount,
 		jobs:        make(chan func()),
+		px:          process.WithTeardown(func() error { return nil }),
 	}
 }
 
 func (bsm *blockstoreManager) start(px process.Process) {
```

Comment:
we can drop this parameter now?

Reply:
I still need to make sure we shutdown the blockstoreManager when we shutdown the parent service (that's what the AddChild call does).